catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/ResourceCacheImplTest.java

```diff
@@ -409,17 +426,29 @@ public void containsSpecificResourceNotInCache() {
     }
 
     private Metacard createMetacard(String sourceId, String metacardId) throws URISyntaxException {
-        
+
         Metacard metacard = generateMetacard();
         metacard.setSourceId(sourceId);
         metacard.setAttribute(new AttributeImpl(Metacard.ID, metacardId));
         return metacard;
     }
 
     private ReliableResource createCachedResource(Metacard metacard) {
-        String fileName = "15bytes.txt";
+        String fileName = "10bytes.txt";
         String productLocation = System.getProperty("user.dir") + "/src/test/resources/" + fileName;
         File rrCachedFile = new File(productLocation);
+        if (!rrCachedFile.exists()) {
```

Comment:
Why would the file not exist? That's the `src/test` directory; if the file isn't already there and then you create it, you've added a file to the repository.

This should definitely not happen.

The `productLocation` should be found by loading the resource and normalizing the path, shouldn't it? Then this wouldn't be an issue.